src/NServiceBus.AcceptanceTests/Performance/ProcessingOptimizations/When_limiting_concurrency_via_both_api_and_config.cs

```diff
@@ -7,18 +7,19 @@ namespace NServiceBus.AcceptanceTests.Config
     using NServiceBus.AcceptanceTests.FakeTransport;
     using NServiceBus.Config;
     using NUnit.Framework;
+    using ScenarioDescriptors;
 
     public class When_limiting_concurrency_via_both_api_and_config : NServiceBusAcceptanceTest
     {
         [Test]
         public async Task Should_throw()
         {
-            var context = await Scenario.Define<Context>()
+            await Scenario.Define<Context>()
                 .WithEndpoint<ThrottledEndpoint>(b => b.CustomConfig(c => c.LimitMessageProcessingConcurrencyTo(10)))
                 .Done(c => c.EndpointsStarted)
+                .Repeat(r => r.For(Transports.AllAvailable.SingleOrDefault(t => t.Key == "FakeTransport")))
```

Comment:
Because .UseTransport<FakeTransport>() seems to be ignored and we get
```
 ----> System.InvalidCastException : Unable to cast object of type 'NServiceBus.AcceptanceTests.FakeTransport.FakeTransport' 
to type 'NServiceBus.AzureServiceBusTransport'```